CONTRIBUTING.md

```diff
@@ -79,25 +79,33 @@ The feature request process is similar to the bug report process but has an extr
 
 Ensure you have Docker, Node.js, yarn, and Golang installed in the correct versions.
 
-Install dependencies with yarn:
+Install dependencies:
 
 ```sh
-$ yarn
+$ make deps-all
 ```
 
 Then build and run the project in a Docker container:
 
 ```sh
-$ yarn start
+$ make dev
 ```
 
-Portainer can now be accessed at <https://localhost:9443>.
+Portainer server can now be accessed at <https://localhost:9443>. and UI dev server runs on <http://localhost:8999>.
+
+if you want to build the project you can run:
+
+```sh
+make build
+```
+
+For additional make commands, run `make help`.
```

Comment:
`make dev` doesn't build the backend, it only starts the container.

The proper build chain is
```bash
make deps-all && make build-server && make dev
```

Reply:
hmm, `make dev-server` should run `build-server`